study-builder/studies/shared/family-history/family-history-child.conf

```diff
@@ -0,0 +1,141 @@
+{
+  include required("../../snippets/activity-general-form.conf"),
+  "studyGuid": ${id.study},
+  STABLE_ID_PREFIX="CHILD"
+  "activityCode": "FAMILY_HISTORY_CHILD",
+  "parentActivityCode": "FAMILY_HISTORY",
+  "createOnParentCreation": false,
+  "canDeleteInstances": true,
+  "displayOrder": 5,
+  "versionTag": "v1",
+  "allowUnauthenticated": false,
+  "listStyleHint": "NONE",
+  "translatedNames": [
+    {
+      "language": "en",
+      "text": """$ddp.answer("CHILD_RELATION", "Child")"""
+    }
+  ],
+  "translatedTitles": [
+    {
+      "language": "en",
+      "text": "Children"
```

Comment:
Again, title should be singular "Child". Also check singular for the "sibling" nested activity.

Reply:
6a66cf22